table/block_based/full_filter_block.cc

```diff
@@ -220,7 +213,7 @@ void FullFilterBlockReader::PrefixesMayMatch(
 void FullFilterBlockReader::MayMatch(
     MultiGetRange* range, bool no_io, const SliceTransform* prefix_extractor,
     BlockCacheLookupContext* lookup_context) const {
-  CachableEntry<BlockContents> filter_block;
+  CachableEntry<FullFilterData> filter_block;
```

Comment:
filter_block -> filter_cache_data?

Reply:
Considering the above, how about `filter_data` ?